### DISCUSSION TITLE
Create View of catchment connection to nodes

### DISCUSSION THREAD

**Discussion by TriformMoritz** [QGEP/QGEP#695] [General]
It would be nice to have a view (a line) between the catchments and the connected nodes (to see where they are connected) like this:
![image](https://user-images.githubusercontent.com/33837122/136010038-e84974b4-b5e2-40d7-9ba4-188a9d439958.png)
They should by a line for each node connected:
- fk_wastewater_networkelement_rw_current
- fk_wastewater_networkelement_ww_current
- fk_wastewater_networkelement_rw_planned
- fk_wastewater_networkelement_ww_planned

Here is what I tried in postgis:

- New view : vw_catchment_connections
- Code (did not work):
SELECT makeline(st_centroid(catchment_area.perimeter_geometry), wastewater_node.situation_geometry)
FROM qgep_od.catchment_area
    LEFT JOIN qgep_od.wastewater_node ON catchment_area.fk_wastewater_networkelement_rw_current ::text = wastewater_node.obj_id::text;  

Could somebody tell me the correct code? And can we implement the view in the next version?

**Comment by urskaufmann** [comment]
There is already the vw_catchment_area_connections in the datamodell. But this view shows only the lines for the current-fk.

I wanted some more attributes and the lines for all 4 fk and I use the views in the zip-file:

[vw_catchment_area_x_connections.zip](https://github.com/QGEP/QGEP/files/7285475/vw_catchment_area_x_connections.zip)

**Comment by TriformMoritz** [comment]
Thank you! 
In which version are these views? I have a 1.5.4 DataModel.

**Comment by urskaufmann** [comment]
1.5.3 (or higher should work)

**Comment by sjib** [comment]
Should this be integrated in the next project file update?
